backend/backendutil/search_test.go

```diff
@@ -10,8 +10,14 @@ import (
 	"github.com/emersion/go-message"
 )
 
+var testInternalDate = time.Unix(1483997966, 0)
+
 var matchTests = []struct {
 	criteria *imap.SearchCriteria
+	seqNum   uint32
+	uid      uint32
+	date     time.Time
+	flags    []string
```

Comment:
Style: alignment issue here

Reply:
Ugh, gofmt seems to be fine with it. How it should be formatted?

Reply:
Oh, if gofmt is fine with that, let's keep it that way

Reply:
Argh, I see: for some reason GitHub displays `fl` as a ligature in one character. Thus `[]string` is misaligned. That's pretty weird.